build-image.sh

```diff
@@ -90,7 +90,7 @@ elif [ "$IMAGE_TYPE" = "orangepipc2" ]; then
 	fi
 fi
 
-dl_dep k3os-rootfs-arm64.tar.gz https://github.com/rancher/k3os/releases/download/v0.9.1/k3os-rootfs-arm64.tar.gz
+dl_dep k3os-rootfs-arm64.tar.gz https://github.com/rancher/k3os/releases/download/v0.10.0/k3os-rootfs-arm64.tar.gz
```

Comment:
How about just using the latest? `https://github.com/rancher/k3os/releases/latest/download/k3os-rootfs-arm64.tar.gz` ?

Reply:
I would prefer using an explicit version, as then we can test new versions before merging and ensure that master always works. However, I would be in favor of supporting an option like `--latest` which takes latest l3os (or perhaps `--version` which can take value 'latest') in case someone wants to use it anyway.

Reply:
Good point, thanks!

Reply:
I agree that I'd like to be able to keep the installs deterministic. One compromise might be to default to `latest` but also allow a switch to call for a specific version.